From a19a51df9e95e36edf949e68581731bd31959081 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@linux.intel.com>
Date: Mon, 13 Apr 2009 21:08:21 +0000
Subject: [tests] Use 'static' when registering enum types

Bug 1393 - Regression tests fails on OpenBSD

When registering a GEnumClass we need to set the GEnumValue array
as static const because g_enum_register_static() does not contain
the word "static" just for fun.

Reported by: Jasper <jasper@humppa.nl>
---
diff --git a/tests/test-serialize-full.c b/tests/test-serialize-full.c
index f549fe1..95925da 100644
--- a/tests/test-serialize-full.c
+++ b/tests/test-serialize-full.c
@@ -92,7 +92,7 @@ test_enum_get_type (void)
 
   if (G_UNLIKELY (e_type == 0))
     {
-      const GEnumValue values[] = {
+      static const GEnumValue values[] = {
         { TEST_ENUM_FOO, "TEST_ENUM_FOO", "foo" },
         { TEST_ENUM_BAR, "TEST_ENUM_BAR", "bar" },
         { TEST_ENUM_BAZ, "TEST_ENUM_BAZ", "baz" },
--
cgit v0.8.2
